flow360/examples/tutorial_bet_disk.py

```diff
@@ -0,0 +1,16 @@
+"""
+tutorialBetDisk meshing example
+"""
+
+from .base_test_case import BaseTestCase
+
+
+class TutorialBETDisk(BaseTestCase):
+    name = "tutorialBetDisk"
+
+    class url:
+        geometry = "local://geometry.csm"
```

Comment:
Same comment as in https://github.com/flexcompute/Flow360/pull/572#discussion_r1856853420
For the geometry file, if the source file's link is provided in the original tutorial, could you try to use the link here to download the input file instead of putting it in the python client repo?

Reply:
Yes, I will use the link here